Missing STRUCT_SIZE on aarch64

When I build for `aarch64-pc-windows-msvc`, I get this:
```
error[E0425]: cannot find value `STRUCT_SIZE` in this scope
   --> C:\Users\task_1559064889\.cargo\registry\src\github.com-1ecc6299db9ec823\remove_dir_all-0.5.1\src\fs.rs:165:54
    |
165 |     let mut data: Vec<u16> = iter::repeat(0u16).take(STRUCT_SIZE/2)
    |                                                      ^^^^^^^^^^^ not found in this scope

error[E0425]: cannot find value `STRUCT_SIZE` in this scope
   --> C:\Users\task_1559064889\.cargo\registry\src\github.com-1ecc6299db9ec823\remove_dir_all-0.5.1\src\fs.rs:179:42
    |
179 |         (*info).FileNameLength = (size - STRUCT_SIZE) as DWORD;
    |                                          ^^^^^^^^^^^ not found in this scope

error: aborting due to 2 previous errors
```
We're missing a target_arch entry for aarch64 at https://github.com/XAMPPRocky/remove_dir_all/blob/ed728d811f19cf12b4649fb9dbb7bf62930ba221/src/fs.rs#L167-L172

--- a/appveyor.yml
+++ b/appveyor.yml
@@ -1,38 +1,53 @@
 environment:
   global:
+    HOST: x86_64-pc-windows-msvc
     RUST_VERSION: stable
     CRATE_NAME: remove_dir_all
   matrix:
     # MinGW
     - TARGET: i686-pc-windows-gnu
+      HOST: i686-pc-windows-gnu
     - TARGET: x86_64-pc-windows-gnu
+      HOST: x86_64-pc-windows-gnu
 
     # MSVC
     - TARGET: i686-pc-windows-msvc
     - TARGET: x86_64-pc-windows-msvc
+    - TARGET: aarch64-pc-windows-msvc
+      APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2017
 
     # Testing other channels
     # Beta channel
     - TARGET: i686-pc-windows-gnu
+      HOST: i686-pc-windows-gnu
       RUST_VERSION: beta
     - TARGET: x86_64-pc-windows-gnu
+      HOST: x86_64-pc-windows-gnu
       RUST_VERSION: beta
 
     - TARGET: i686-pc-windows-msvc
       RUST_VERSION: beta
     - TARGET: x86_64-pc-windows-msvc
       RUST_VERSION: beta
+    - TARGET: aarch64-pc-windows-msvc
+      RUST_VERSION: beta
+      APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2017
 
     # Nightly channel
     - TARGET: i686-pc-windows-gnu
+      HOST: i686-pc-windows-gnu
       RUST_VERSION: nightly
     - TARGET: x86_64-pc-windows-gnu
+      HOST: x86_64-pc-windows-gnu
       RUST_VERSION: nightly
 
     - TARGET: i686-pc-windows-msvc
       RUST_VERSION: nightly
     - TARGET: x86_64-pc-windows-msvc
       RUST_VERSION: nightly
+    - TARGET: aarch64-pc-windows-msvc
+      RUST_VERSION: nightly
+      APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2017
 
 matrix:
   allow_failures:
@@ -46,17 +61,22 @@ install:
         $Env:PATH += ';C:\msys64\mingw32\bin'
       }
   - curl -sSf -o rustup-init.exe https://win.rustup.rs/
-  - rustup-init.exe -y --default-host %TARGET% --default-toolchain %RUST_VERSION%
+  - rustup-init.exe -y --default-host %HOST% --default-toolchain %RUST_VERSION%
   - set PATH=%PATH%;C:\Users\appveyor\.cargo\bin
+  - rustup target install %TARGET%
   - rustc -Vv
   - cargo -V
+build_script:
+  - cargo build --target %TARGET% &&
+    cargo build --target %TARGET% --release
 # TODO This is the "test phase", tweak it as you see fit
 test_script:
-  # we don't run the "test phase" when doing deploys
+  # we don't run the "test phase" when doing deploys, nor do we run it
+  # when the target is aarch64-pc-windows-msvc, since the Appveyor image
+  # can't execute that target's executables
   - if [%APPVEYOR_REPO_TAG%]==[false] (
-      cargo build --target %TARGET% &&
-      cargo build --target %TARGET% --release &&
-      cargo test --target %TARGET% &&
-      cargo test --target %TARGET% --release
+      if not [%TARGET%]==[aarch64-pc-windows-msvc] (
+        cargo test --target %TARGET% &&
+        cargo test --target %TARGET% --release
+      )
     )
-build: false

--- a/src/fs.rs
+++ b/src/fs.rs
@@ -24,9 +24,11 @@ struct RmdirContext<'a> {
 /// ```rust
 /// extern crate remove_dir_all;
 ///
+/// use std::fs;
 /// use remove_dir_all::*;
 ///
 /// fn main() {
+///     fs::create_dir("./temp/").unwrap();
 ///     remove_dir_all("./temp/").unwrap();
 /// }
 /// ```
@@ -166,9 +168,9 @@ fn move_item(file: &File, ctx: &mut RmdirContext) -> io::Result<()> {
 fn rename(file: &File, new: &Path, replace: bool) -> io::Result<()> {
     // &self must be opened with DELETE permission
     use std::iter;
-    #[cfg(target_arch = "x86")]
+    #[cfg(target_pointer_width = "32")]
     const STRUCT_SIZE: usize = 12;
-    #[cfg(target_arch = "x86_64")]
+    #[cfg(target_pointer_width = "64")]
     const STRUCT_SIZE: usize = 20;
 
     // FIXME: check for internal NULs in 'new'

